### PR TITLE
Build updates.

### PR DESCRIPTION
**Pull request by kshakir**
More info generated during scalatest.
Excluding scalacheck from the build.
Fixed README references to old hyphenated sbt key names.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,12 +19,12 @@ Tag names are all lower case, so a test named "tagFoo" has a tag "tagfoo".
...
 To run only those tests which have been tagged with a specified tag `tagFoo`:
 ```
-sbt "test-only * -- -n tagfoo"
+sbt "testOnly * -- -n tagfoo"
 ```
 
 Or to instead exclude all tests which have been tagged with a specified tag `tagFoo`:
 ```
-sbt "test-only * -- -l tagfoo"
+sbt "testOnly * -- -l tagfoo"
```
Comment (mcovarr) [inline README.md:27]: why these changes?

File: build.sbt [hunk 2]
```diff
@@ -17,10 +17,16 @@ val akkaV = "2.4.11" // Note: akka-http branches from akkaV after 2.4.11
...
   "com.typesafe.akka" %% "akka-http-spray-json-experimental" % akkaV,
-  "org.scalatest" %% "scalatest" % "2.2.6" % Test,
-  "com.github.pathikrit" %% "better-files" % "2.13.0"
+  "com.github.pathikrit" %% "better-files" % "2.13.0",
+  //---------- Test libraries -------------------//
+  "org.scalatest" %% "scalatest" % "3.0.1" % Test,
+  "org.pegdown" % "pegdown" % "1.6.0" % Test
 )
+
+testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-oDSI", "-h", "target/test-reports")
```
Comment (mcovarr) [inline build.sbt:32]: what is this doing?

**Review by mcovarr — APPROVED** [review]

**Comment by mcovarr** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/163/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: README.md (hunk 1 above)
Comment (kshakir) [reply README.md:27]: Via http://www.scala-sbt.org/0.13/docs/ChangeSummary_0.13.0.html#camelCase+Key+names:

> The convention for key names is now camelCase only instead of camelCase for Scala identifiers and hyphenated, lower-case on the command line. camelCase is accepted for existing hyphenated key names and the hyphenated form will still be accepted on the command line for those existing tasks and settings declared with hyphenated names. Only camelCase will be shown for tab completion, however.

File: build.sbt (hunk 2 above)
Comment (kshakir) [reply build.sbt:32]: OCD answer: It's syncing up the reporting to be the same as cromwell.

Technical answer via [docs](http://www.scalatest.org/user_guide/using_the_runner):
- `-oD` - show all durations
- `-oS` - show short stack traces
- `-oI` - after each sbt project finishes, of which wdl4s only has one, reprint a summary of all the failed tests
- `-h target/test-reports` - create an html report of the test results

File: README.md (hunk 1 above)
Comment (cjllanwarne) [reply README.md:27]: Progress gratia progressis? :p

**Review by cjllanwarne — APPROVED** [review]

**Comment by cjllanwarne** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/163/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)
